packages/react/src/common/useFocusByKeyboard.tsx

```diff
@@ -24,21 +24,18 @@ const FOCUSABLE_ELEMENTS = [
   'select:not([disabled])',
```

Comment:
Maybe not for now, but: do we want to document this hook? We expose it to our users already

Reply:
Ah, sorry, I was wrong. It is a function in `index.esm.js` and `index.cjs.js`, but it's not exported, so you probably can't reach it?

Reply:
Yes, nothing seems to indicate it’s public.